github/resource_github_actions_runner_group.go

```diff
@@ -0,0 +1,232 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/google/go-github/v36/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v38/github"
```

We just bumped to v38, so lets update this.